cmd/state-tray/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ActiveState/cli/cmd/state-tray/internal/open"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/getlantern/systray"
+	"github.com/gobuffalo/packr"
+)
+
+func main() {
+	systray.Run(onReady, onExit)
+}
+
+func onReady() {
```

Comment:
None of the user facing strings here are localized.

Reply:
Whoops, done now.